tests/test_aircon.py

```diff
@@ -110,6 +113,7 @@ async def test_attributes(
     assert aircon.get_quiet_mode() is True
     await aircon.disconnect()
 
+    appliance_http_client_mock.close_session()
```

Comment:
```suggestion
    await appliance_http_client_mock.close_session()
```
I just noticed that this needs to be awaited. Can you please also update the others? 
You can see the warnings for this PR here: https://github.com/abmantis/whirlpool-sixth-sense/actions/runs/3699881981/jobs/6267722655#step:5:132

Reply:
I can see them at the link - what's strange is running the tests locally in Visual Studio are not throwing these... 
I'll have to dig into my local environment to see why.

It appears that running pytest from the command line does show the errors, but in Visual Studio testing "passes".  I'll have to remember to use the command line (for now)

Reply:
Yeah, the tests pass anyway. Those are just warnings.